app/uk/gov/hmrc/tariffclassificationfrontend/views/reassign_queue_case.scala.html

```diff
@@ -38,7 +38,7 @@ <h3 class="heading-large mt-0">Move this case back to a queue</h3>
                     queues,
                     "reassign-case",
                     ReassignCaseController.reassignCase(c.reference),
-                    CaseController.activityDetails(c.reference)
+                    request.headers.get("Referer").getOrElse(CaseController.applicationDetails(c.reference).url)
```

Comment:
this causes an issue if you do not select the radio button and press `Submit` before `Cancel`

Reply:
This must be part of Steves PR. If you fix it in there it will fix it here

Reply:
Cant imagine this actually works to be honest

Reply:
If you refresh the page it wont work